locale/fi/servicenavigation.json

```diff
@@ -0,0 +1,70 @@
+{
+  "title": "Palvelunavigaatio (ServiceNavigation)",
+  "intro": "Palvelunavigaatiota käytetään jonkin sivusto-osion eri alasivujen/-näkymien välillä liikkumiseen.",
```

Comment:
I'd use a written form instead of the "alasivujen/-näkymien" style to keep things more readable (and screen reader friendly).

Reply:
Check!